### ISSUE TITLE
the directory structure of RAVDESS and CMU_MOSEI

### ISSUE THREAD

**Issue by jianing-zhang** [ricfrr/mpuc-mer#4]
Could you please give the directory structure and content of RAVDESS and CMU-MOSEI? Thank you

**Comment by Knivacke** [comment]
Hey. Sorry to bother you. Did you manage to figure it out?

**Comment by jianing-zhang** [comment]
> Hey. Sorry to bother you. Did you manage to figure it out?
No, I still don't know. But you can email me and we can discuss the information. My email is [zjn0861@foxmail.com](mailto:zjn0861@foxmail.com)
